android_automation/test_cases/home_test.py

```diff
@@ -278,21 +278,29 @@ def test_home_banner(self, wd, test_result='PASS', error_texts=[], img_src='', w
             aalc(wd, 'HOME')
             print("홈 탭 선택")
 
-            # 보강시나리오
-            # aalc(wd, 'life_tab')
-            # print("라이프 탭 선택")
-            # sleep(2)
-            # 라이프탭 아이디 설정 후 추가 필요
-            # aalc(wd, 'com.the29cm.app29cm:id/closeIcon')
-            aalc(wd, 'life_tab')
-            print("라이프 탭 해제")
-            sleep(1)
+            # 디폴트 선택 화면 확인
+            try:
+                top_tabs = aal(wd, 'com.the29cm.app29cm:id/tabs')
+                culture_tab = aal(top_tabs,
+                                  '//android.view.View/android.view.View/android.view.View[@index=4]')
```

Comment:
```suggestion
                                  '//android.view.View[@index=4]')
```
이렇게 해도 찾아질 것 같은데 한번 해보시겠어요? 계층 구조는 길면 길수록 깨지기 쉽습니다

Reply:
넵 전달 주신 코드로도 동작하여 해당 코드로 수정했습니다. 감사합니다.